NumLib/Extrapolation/LocalLinearLeastSquaresExtrapolator.h

```diff
@@ -10,7 +10,7 @@
 #ifndef NUMLIB_LOCAL_LLSQ_EXTRAPOLATOR_H
 #define NUMLIB_LOCAL_LLSQ_EXTRAPOLATOR_H
 
-#include "AssemblerLib/GlobalMatrixProviders.h"
+#include "NumLib/DOF/GlobalMatrixProviders.h"
```

Comment:
I'd leave it in `MathLib`, since it only copes with Mathlib stuff. Or is that impossible?
In particular I wouldn't put it into the DOF directory, because it merely provides matrices and vectors.